.github/workflows/php.yml

```diff
@@ -0,0 +1,20 @@
+name: Rector Code Review
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    #- name: Validate composer.json and composer.lock
+    #  run: composer validate
```

Comment:
It seem this is debug left over. Should be removed

Reply:
Sure, is removed. This is a default action added by GitHub, which failed because some license resp. private as well as name and description fields are missing in composer.json.